src/benchq/problem_ingestion/molecule_instance_generation.py

```diff
@@ -293,7 +211,7 @@ def _get_molecular_data(self):
         )
 
         molecule, mean_field_object = self._run_pyscf()
-        molecular_data.n_orbitals = int(molecule.nao_nr())
+        molecular_data.n_orbitals = int(molecule.nao)
```

Comment:
I am not sure .nao always gives the correct number of contracted GTOs - I would rather keep .nao_nr(). According to pyscf documentation:

nao_nr(cart=None)(https://pyscf.org/pyscf_api_docs/pyscf.gto.html#pyscf.gto.mole.Mole.nao_nr)
Total number of contracted GTOs for the given [Mole](https://pyscf.org/pyscf_api_docs/pyscf.gto.html#pyscf.gto.mole.Mole) object